src/Service/AbstractService.php

```diff
@@ -4,6 +4,8 @@
 
 use Framework\Contracts\SessionInterface;
 use Framework\Http\Request;
+use QuizApp\Entity\FiltersForEntity;
+use ReallyOrm\Entity\EntityInterface;
```

Comment:
nu vad sa fie folosite clasele astea pe undeva in AbstractService. Daca nu sunt folosite, poti sterge use-urile